launcher/minecraft/mod/ModFolderModel.cpp

```diff
@@ -171,6 +179,8 @@ QVariant ModFolderModel::headerData(int section, [[maybe_unused]] Qt::Orientatio
                     return tr("The date and time this mod was last changed (or added).");
                 case ProviderColumn:
                     return tr("Where the mod was downloaded from.");
+                case SizeColumn:
+                    return tr("The size of the resource.");
```

Comment:
```suggestion
                    return tr("The size of the mod.");
```

same applies elsewhere